storage/platform.go

```diff
@@ -0,0 +1,48 @@
+package storage
+
+import (
+	"time"
+
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+)
+
+const (
+	EntityDetails        = "ATLAS_PLATFORM_DETAILS"
+	entityDetailsUpdated = "ATLAS_PLATFORM_DETAILS_UPDATED"
+	allowedCachingPeriod = 86400 // 24 hours to UNIX time
+)
+
+func (s *Storage) SaveCachedDetails(coin string, details *blockatlas.StakingDetails) error {
+	return s.AddHM(EntityDetails, createHashMap(coin, ""), details)
```

Comment:
you can use only the coin name here

```suggestion
	return s.AddHM(EntityDetails, coin, details)
```

https://github.com/trustwallet/blockatlas/blob/master/storage/market.go#L73

Reply:
Nice!